lib/eslint.js

```diff
@@ -70,6 +70,25 @@ function parseBooleanConfig(string, comment) {
     return items;
 }
 
+var VALID_LEVELS = [0, 1, 2, "off", "warn", "error"];
+
+/**
+ * Checks whether all rules' level is valid or not.
+ *
+ * @param {object} config - A config object to check.
+ * @returns {boolean} `true` if all rules' level is valid.
+ */
+function levelsAreValid(config) {
```

Comment:
Maybe `isInlineConfigValid`?

Also, maybe create `ConfigOps.isValidSeverity()` to keep that logic in one place?